src/js/components/location/LocationChooser/LocationChooserButton/LocationChooserButton.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { Tooltip } from 'react-tippy';
+
+import Translate from 'utils/Translate';
+
+import './LocationChooserButton.scss';
+
```

Comment:
shouldn't we use absolute imports everywhere? e.g. `components/LocationChooserButton/LocationChooseButton.scss`